### PR TITLE
(DOCSP-29570): Kotlin: Update PersistedName annotation for Class support

### PR DESCRIPTION
**Pull request by dacharyc**
### Jira

- https://jira.mongodb.org/browse/DOCSP-29570

### Staged Changes

- [Model Data/Property Annotations](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-29570/sdk/kotlin/realm-database/schemas/property-annotations/#map-a-property-or-class-to-a-different-name): Updates to the "Map a Property (Or Class) to a Different Name" section

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/kotlin/realm-database/schemas/property-annotations.txt [hunk 1]
```diff
@@ -81,38 +81,60 @@ You can create a primary key with any of the following types:
...
-persisted name used to store values in Realm Database:
+Note that migrations must use the persisted class or property name, 
+and any schema errors reported also use the persisted name. 
 
-.. literalinclude:: /examples/generated/kotlin/SchemaTest.snippet.declare-properties.kt
-  :language: kotlin
-  :emphasize-lines: 15-16
+.. tabs::
+
+   .. tab:: Remap a Class
```
Comment (MongoCaleb) [inline source/sdk/kotlin/realm-database/schemas/property-annotations.txt:112]: Can we add a _ref to a tab? If so, that will be useful for linking.

File: source/sdk/kotlin/realm-database/schemas/property-annotations.txt [hunk 2]
```diff
@@ -81,38 +81,60 @@ You can create a primary key with any of the following types:
...
+      :tabid: class-persisted-name
+
+      In this example, ``Cat`` is the Kotlin class name used in the code 
+      throughout the project to perform CRUD operations, and ``Feline`` is the
+      persisted name to used to store objects in Realm Database:
+
+      .. literalinclude:: /examples/generated/kotlin/SchemaTest.snippet.class-persisted-name.kt
+        :language: kotlin
+
+      Queries on class names remapped in this way must use the persisted name,
```
Comment (MongoCaleb) [inline source/sdk/kotlin/realm-database/schemas/property-annotations.txt:122]: This feels important and might get lost. I think it belongs in a ..note or, perhaps, and ..important.

**Review by MongoCaleb — APPROVED** [review]
Approving with 2 suggestions/questions.

File: source/sdk/kotlin/realm-database/schemas/property-annotations.txt (hunk 1 above)
Comment (dacharyc) [reply source/sdk/kotlin/realm-database/schemas/property-annotations.txt:112]: Alas, we cannot add links directly to tabs.

**Pull request merged**